addons/account/models/account_journal.py

```diff
@@ -800,11 +800,6 @@ def name_get(self):
             res += [(journal.id, name)]
         return res
 
-    def action_archive(self):
```

Comment:
For stable policy, avoid deleting the method, you can remove the check and add a comment like `# TODO: remove in master`